master/buildbot/config/master.py

```diff
@@ -397,7 +397,7 @@ def copy_str_or_callable_param(name):
             try:
                 import brotli  # pylint: disable=import-outside-toplevel
 
-                [brotli]
+                _ = brotli
```

Comment:
Also my lsp recognizes this better as an unused value. Also looks less weird to me as a reader. First time seeing this it looked like SyntaxError. Also doesn't allocate a list.